tuxemon/core/db.py

```diff
@@ -86,11 +75,22 @@ def load(self, directory="all"):
 
         """
 
+        self.database = {
+            "item": {},
+            "monster": {},
+            "npc": {},
+            "technique": {},
+            "encounter": {},
+            "inventory": {},
+            "environment": {},
+        }
+
+        self.path = prepare.fetch("db")
         if directory == "all":
             self.load_json("item")
             self.load_json("monster")
-            self.load_json("technique")
             self.load_json("npc")
+            self.load_json("technique")
```

Comment:
why is this moved?

Reply:
I noticed it wasn't in the same order as the database properties declared in init. "technique" comes after "npc" there, but below it was above it... I figured this would be more clean just in case.